src/Tomcat/TomcatServer.ts

```diff
@@ -4,8 +4,12 @@ import * as path from "path";
 import * as vscode from "vscode";
 import { Utility } from "../Utility";
 
-export class TomcatServer {
+export class TomcatServer implements vscode.QuickPickItem {
```

Comment:
rename TomcatServer to TomcatServerItem

Reply:
Mostly use of TomcatServer is its server feature. Or, we could make a independent class TomcatServerQuickPickItem, but it seems not that necessary, just use TomcatServer.